springwolf-examples/springwolf-cloud-stream-example/build.gradle

```diff
@@ -74,6 +74,9 @@ java {
 }
 
 test {
+    minHeapSize = "128m" // initial heap size
+    maxHeapSize = "1024m" // maximum heap size
```

Comment:
Locally, we get an OutOfMemory during test execution in `springwolf-cloud-stream-example`. So we defined a maxHeapSize which was sufficient.